docs/Contributing/Building-Fleet.md

```diff
@@ -158,14 +158,24 @@ To start the Fleet server backed by the Docker development infrastructure, run t
 ./build/fleet serve --dev
 ```
 
-The server is accessible by default at [https://localhost:8080](https://localhost:8080). Note that `--dev` requires the use of `make generate-dev` as the server will not use bundled assets in this mode (you may see an error mentioning a template not found when visiting the website otherwise).
+### Developing the Fleet UI
+
+When the Fleet server is running the Fleet UI is accessible by default at
+[https://localhost:8080](https://localhost:8080).
+
+> Note that `./build/fleet serve --dev` requires the use of `make generate-dev` as the server will not use bundled assets in this mode (you may see an error mentioning a template not found when visiting the website otherwise).
```

Comment:
```suggestion
> Note that `./build/fleet serve --dev` requires the use of `make generate-dev` because the server will not use bundled assets in this mode. (You may see an error mentioning a template not found when visiting the website otherwise.)
```